README.md

```diff
@@ -31,21 +37,25 @@ Hudi manages the storage of large analytical datasets on DFS (Cloud stores, HDFS
 Hudi provides the ability to query via three types of views:
  * **Read Optimized View** - Provides excellent snapshot query performance via purely columnar storage (e.g. [Parquet](https://parquet.apache.org/))
  * **Incremental View** - Provides a change stream with records inserted or updated after a point in time.
- * **Real-time View** - Provides snapshot queries on real-time data, using a combination of columnar & row-based storage (e.g Parquet + [Avro](http://avro.apache.org/docs/current/mr.html))
+ * **Real-time View** - Provides snapshot queries on real-time data, using a combination of columnar & row-based storage (e.g [Parquet](https://parquet.apache.org/) + [Avro](http://avro.apache.org/docs/current/mr.html))
 
 Learn more about Hudi at [https://hudi.apache.org](https://hudi.apache.org)
 
-### Building Apache Hudi from source {#building-hudi}
+## Building Apache Hudi from source
+
+Prerequisites for building Hudi:
```

Comment:
why `Apache Hudi` above and just `Hudi` here? :). I think we can standardize on one?

Reply:
right